cmstestsuite/TestCleanCheckout.py

```diff
@@ -77,7 +77,8 @@ def setup_cms():
     os.environ["PYTHONPATH"] = "%(TEST_DIR)s" % CONFIG
 
     info("Building cms.")
-    sh("./setup.py build")
+    sh("./prerequisites.py compile_l10n")
+    sh("./prerequisites.py compile_isolate")
```

Comment:
I don't know if it's worth to pollute the setuptools script with the compilation of isolate and l10n files, because:

1. After building, isolate and l10n files will then get installed by prerequisites.py (because they are based on a different path than the one used by the cms python package).
2. Isolate could eventually be separately packaged and installed.
3. Having server-side l10n is not generally a "future-proof" idea either, since with RESTful APIs it's generally better to do client-side localization (see [here](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Intl)), so there is the possibility of switching to JS localization files (which are easily installed in the cms python package without the need of compiling them).